### ISSUE TITLE
MaxListenersExceededWarning encountered

### ISSUE THREAD

**Issue by jaydenseric** [mike-marcacci/fs-capacitor#1]
When fs-capacity is used frequently in the same process, it can trigger `MaxListenersExceededWarning` (best debugged with the `--trace-warnings` Node.js flag):

```
(node:72923) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added. Use emitter.setMaxListeners() to increase limit
    at _addListener (events.js:243:17)
    at process.addListener (events.js:259:10)
    at _fs.default.open (/Users/jaydenseric/Sites/apollo-upload-server/node_modules/fs-capacitor/lib/index.js:140:17)
    at FSReqWrap.oncomplete (fs.js:145:20)
(node:72923) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added. Use emitter.setMaxListeners() to increase limit
    at _addListener (events.js:243:17)
    at process.addListener (events.js:259:10)
    at _fs.default.open (/Users/jaydenseric/Sites/apollo-upload-server/node_modules/fs-capacitor/lib/index.js:141:17)
    at FSReqWrap.oncomplete (fs.js:145:20)
```

See https://travis-ci.org/jaydenseric/apollo-upload-server/jobs/405666803#L2191.

Here is where the relevant listeners are attached: https://github.com/mike-marcacci/fs-capacitor/blob/v0.0.3/src/index.mjs#L142

**Comment by mike-marcacci** [comment]
This should now be fixed on master and in `v1.0.0-beta.*`: https://travis-ci.org/jaydenseric/apollo-upload-server/jobs/406965783#L2203

**Comment by mike-marcacci** [comment]
Fixed in `v1.0.0`
